lua/telescope/mappings.lua

```diff
@@ -191,7 +191,11 @@ mappings.apply_keymap = function(prompt_bufnr, attach_mappings, buffer_keymap)
       local key_bind_internal = a.nvim_replace_termcodes(key_bind, true, true, true)
       if not applied_mappings[mode][key_bind_internal] then
         applied_mappings[mode][key_bind_internal] = true
-        telescope_map(prompt_bufnr, mode, key_bind, key_func)
+        if type(key_func) == "string" then
```

Comment:
maybe we should just actually do this check above at  the top of the function, inside of the map function we create on line 104 -- that way any time someone uses a string, it defaults to searching `actions`.